cmd/cmd_test.go

```diff
@@ -208,3 +212,152 @@ func TestCommandsIntegration(t *testing.T) {
 		}()
 	}
 }
+
+// Test that saving a dataset with a relative body path works, and validate the contents of that
+// body match what was given to the save command.
+func TestSaveRelativeBodyPath(t *testing.T) {
+	if err := confirmQriNotRunning(); err != nil {
+		t.Skip(err.Error())
+	}
+
+	repo := NewTestRepoRoot("qri_test_save_relative_body")
+	defer repo.Delete()
+
+	// TODO: If TestRepoRoot is moved to a different package, pass it an a parameter to this
+	// function.
+	cmdR := repo.CreateCommandRunner()
+	_, err := executeCommand(cmdR, "qri save --file=testdata/movies/ds_ten.yaml me/test_movies")
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	// Read body from the dataset that was saved.
+	dsPath := repo.GetPathForDataset(0)
+	actualBody := repo.ReadBodyFromIPFS(dsPath + "/data.csv")
+
+	// Read the body from the testdata input file.
+	f, _ := os.Open("testdata/movies/body_ten.csv")
+	expectBytes, _ := ioutil.ReadAll(f)
+	expectBody := string(expectBytes)
+
+	// Make sure they match.
+	if actualBody != expectBody {
+		t.Errorf("error reading body, expect \"%s\", actual \"%s\"", actualBody, expectBody)
+	}
+}
+
+// TODO: Perhaps this utility should move to a lower package, and be used as a way to validate the
+// bodies of dataset in more of our test case. That would require extracting some parts out, like
+// pathFactory, which would probably necessitate the pathFactory taking the testRepoRoot as a
+// parameter to its constructor.
+
+// TestRepoRoot stores paths to a test repo.
+type TestRepoRoot struct {
+	rootPath    string
+	ipfsPath    string
+	qriPath     string
+	pathFactory PathFactory
+	testCrypto  gen.CryptoGenerator
+}
+
+// NewTestRepoRoot constructs the test repo and initializes everything as cheaply as possible.
+func NewTestRepoRoot(prefix string) TestRepoRoot {
```

Comment:
I think for stuff that's purely intended for use with tests, it's fine to accept a `testing.T` and call `t.Fatal` instead of panicing, prevents whole test suite from blowing up.
```suggestion
func NewTestRepoRoot(t *testing.T, prefix string) TestRepoRoot {
```